specification/purview/resource-manager/readme.go.md

```diff
@@ -14,16 +14,6 @@ go:
 ``` yaml $(go) && $(multiapi)
 batch:
   - tag: package-2021-07-01
-  - tag: package-2020-12-01-preview
-```
-
-### Tag: package-2020-12-01-preview and go
-
-These settings apply only when `--tag=package-2020-12-01-preview --go` is specified on the command line.
-Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
-
-``` yaml $(tag) == 'package-2020-12-01-preview' && $(go)
-output-folder: $(go-sdk-folder)/services/preview/purview/mgmt/2020-12-01-preview/$(namespace)
```

Comment:
Please do not remove this if the corresponding swagger is not removed

Reply:
Issue we are facing is without this the sdk generation is broken.This is an old preview api where we already shipped the sdk to customers.The two options we are facing

1. keep the go sdk for preview api version - this generates a breaking change 

2. remove go sdk definition - this generates a breaking change

this change does not target this preview version, for this PR is there a way to leave the go configuration as is and to ignore it to check in our change?